File: PHP PowerPoint Examples/TextApiDemo/getPptSlideTextItems.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\TextApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TextApi($configuration);

$name = "SlideTextItems.pptx";
$withEmpty = 'true'; //the default value is false
$slideIndex = 1;
$password = null;
$folder = "Powerpoint";
$storage = null;
$result = $apiInstance->getPptSlideTextItems($name, $slideIndex, $withEmpty, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/TextApiDemo/getPptSlideTextItems.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.text_api.TextApi(configuration)

#support file format: .pptx/.ppt/.pptm/.pps/.ppsx/.pot/.potx/.odp/
fileName = "SlideTextItems.pptx"
password = ""
storage = ""
#support :flase/true
with_empty = "false"
slideIndex = 1
inputFolder = "input"
result = api.get_ppt_slide_text_items(fileName, with_empty=with_empty, slide_index=slideIndex,password=password, storage=storage, folder=inputFolder)